addons/website/tests/test_get_current_website.py

```diff
@@ -134,3 +137,34 @@ def test_02_signup_user_website_id(self):
 
         user = self.env['res.users'].create({'website_id': website.id, 'login': 'sad@mail.com', 'name': 'Hope Fully'})
         self.assertTrue(user.website_id == user.partner_id.website_id == website)
+
+
+@tagged('post_install', '-at_install')
+class TestRecursionIssue(HttpCase):
+    def test_recursive_current_website(self):
+        Rule = self.env['ir.rule']
+        Rule.create({
+            'name': 'Recursion',
+            'model_id': self.env.ref('website.model_website').id,
+            'domain_force': [(1, '=', 1)],
+            'groups': [],
+        })
+
+        self.session = http.root.session_store.new()
+        http.root.session_store.save(self.session)
+        website = self.env.ref('website.default_website')
+        login_request = requests.post(website.get_base_url() + "/web/login", {
+            'csrf_token': http.WebRequest.csrf_token(self),
+            'login': 'admin',
+            'password': 'admin',
+            'redirect': '',
+        }, cookies={
+            'session_id': self.session.sid,
+            'frontend_lang': 'en_US',
+            'tz': 'Europe/Brussels',
+            'cids': '1',
+        }, timeout=60)
```

Comment:
Can't we reduce it a bit? Is there a reason to have chosen 60?

Reply:
Changed to 5